setup.py

```diff
@@ -36,7 +36,7 @@ def run(self):
 
 
 setup(name='gemd',
-      version='0.7.0',
+      version='0.7.1',
```

Comment:
Since you changed a class name, I think this should have been a minor version bump.  Not worth back-porting; just an fyi.